src/components/pages/order/Main/Menu.jsx

```diff
@@ -50,24 +52,31 @@ export default function Menu() {
   };
 
   return (
-    <MenuStyled className="menu">
+    <TransitionGroup component={MenuStyled} className="menu">
       {menu.map(({ id, title, imageSource, price }) => {
         return (
-          <Card
+          <CSSTransition
+            appear
+            classNames={"card-menu"}
             key={id}
-            title={title}
-            imageSource={imageSource ? imageSource : DEFAULT_IMAGE}
-            leftDescription={formatPrice(price)}
-            hasDeleteButton={isAdmin}
-            onDelete={(event) => handleOnDelete(id, event)}
-            onSelect={() => handleSelectCard(id)}
-            ishoverable={isAdmin}
-            isSelected={checkProductIsClicked(id, productSelected)}
-            handleAddProduct={(event) => handleAddProduct(event, id)}
-          />
+            timeout={{ enter: 1000, exit: 300 }}
+          >
+            <Card
+              key={id}
+              title={title}
+              imageSource={imageSource ? imageSource : DEFAULT_IMAGE}
+              leftDescription={formatPrice(price)}
+              hasDeleteButton={isAdmin}
+              onDelete={(event) => handleOnDelete(id, event)}
+              onSelect={() => handleSelectCard(id)}
+              ishoverable={isAdmin}
```

Comment:
```suggestion
              $isHoverable={isAdmin}
```
il faut rajouter le symbole "$" pour que le HTML en bout de course ne soit pas choqué par cet attribut. C'est comme ça que tu te débarrasseras du message de warning dans la console de ton inspecteur d'éléments.